AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/InviteController.cs

```diff
@@ -0,0 +1,154 @@
+                CampaignId = campaignId,
+            });
+        }
+
+        // GET: Admin/Invite/SendEventManagerInvite/5
+        [Route("[area]/[controller]/[action]/{eventId:int}")]
+        public async Task<IActionResult> SendEventManagerInvite(int eventId)
+        {
+            var @event = await _mediator.SendAsync(new EventByEventIdQuery { EventId = eventId });
+            if (@event == null) return NotFound();
+
+            if (!IsUserAuthorizedToSendInvite(@event.Campaign.ManagingOrganizationId))
+            {
+                return Unauthorized();
+            }
+
+            return View("Send", new InviteViewModel
+            {
+                InviteType = InviteType.EventManagerInvite,
+                EventId = eventId,
+            });
+        }
+
+        // POST: Admin/Invite/SendCampaignManagerInvite/5
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        [Route("[area]/[controller]/[action]/{campaignId:int}")]
+        public async Task<IActionResult> SendCampaignManagerInvite(int campaignId, InviteViewModel invite)
+        {
+            if (invite == null)
+            {
+                return BadRequest();
+            }
+
+            if (ModelState.IsValid)
+            {
+                var campaign = await _mediator.SendAsync(new CampaignByCampaignIdQuery { CampaignId = invite.CampaignId });
+
+                if (campaign == null)
+                {
+                    return BadRequest();
+                }
+
+                if (!IsUserAuthorizedToSendInvite(campaign.ManagingOrganizationId))
+                {
+                    return Unauthorized();
+                }
+
+                var user = await _userManager.GetUserAsync(User);
+                await _mediator.SendAsync(new CreateInviteCommand { Invite = invite, UserId = user?.Id });
+
+                return RedirectToAction(actionName: "Details", controllerName: "Campaign", routeValues: new { area = "Admin", id = invite.CampaignId });
+            }
+
+            return View("Send", invite);
+        }
+
+        // POST: Admin/Invite/SendEventManagerInvite/5
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        [Route("[area]/[controller]/[action]/{eventId:int}")]
+        public async Task<IActionResult> SendEventManagerInvite(int eventId, InviteViewModel invite)
+        {
+            if (invite == null)
+            {
+                return BadRequest();
+            }
+
+            if (ModelState.IsValid)
+            {
+                var @event = await _mediator.SendAsync(new EventByEventIdQuery { EventId = invite.EventId });
+                if (@event == null)
+                {
+                    return BadRequest();
+                }
+
+                if (!IsUserAuthorizedToSendInvite(@event.Campaign.ManagingOrganizationId))
+                {
+                    return Unauthorized();
+                }
+
+                var user = await _userManager.GetUserAsync(User);
```

Comment:
same feedback for this POST action method, remove all the code above here except for the ModelState.IsValid check